### PR TITLE
Remove redux and calls to it.

### PR DESCRIPTION
**Pull request by qwangd2l**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,14 +35,12 @@
   "dependencies": {
     "@brightspace-ui/core": "^1",
     "@chaitin/querystring": "^1.1.0",
-    "@polymer/polymer": "3.2.0",
+    "@polymer/polymer": "^3.2.0",
```
Comment (ktonon) [inline package.json]: Intentional?

File: src/d2l-content-store.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ class D2lContentStore extends DependencyProvider(LitElement) {
 		// Dynamically importing d2l-content-store-app
 		// to avoid "process is not defined" error caused by importing
 		// redux at the top level.
-		window.process = { env: { NODE_ENV: 'production' } };
+		// window.process = { env: { NODE_ENV: 'production' } };
```
Comment (ktonon) [inline src/d2l-content-store.js]: Maybe remove this entire part?

File: package.json (hunk 1 above)
Comment (qwangd2l) [reply package.json]: > Intentional?

Yeah, this is something I noticed when making the PR to BSI.
BSI uses `^3.2.0` and if we keep it `3.2.0`, it'll install another version of polymer separately for our component. Should be safe to make it the same.

**Comment by ktonon** [comment]
Are we keeping the actions and reducers for later conversion to mobx?

**Comment by qwangd2l** [comment]
> Are we keeping the actions and reducers for later conversion to mobx?

Most likely mobx will not use actions and reducers but we need to figure that out.

**Pull request merged**